### PR TITLE
chore(ci): add system tables/columns benchmark cloud

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Summary

add system tables/columns benchmark cloud

- Fixes #[Link the issue here]

## Tests

- [ ] Unit Test
- [ ] Logic Test
- [x] Benchmark Test
- [ ] No Test  - _Explain why_

## Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/14555)

### CODE REVIEW

File: benchmark/clickbench/internal/load.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import os
+import sys
+from datetime import datetime
+import mysql.connector
+import time
+
+
+db = mysql.connector.connect(
+    host="127.0.0.1", user="root", passwd="root", port="3307", buffered=True
```
Comment (sundy-li) [inline benchmark/clickbench/internal/load.py:11]: why not use bendsql? let's get rid of mysql.

File: .github/workflows/reuse.benchmark.yml [hunk 2]
```diff
@@ -100,6 +100,8 @@ jobs:
           - { dataset: hits, size: Large }
           - { dataset: tpch, size: Small }
           - { dataset: tpch, size: Large }
+          - { dataset: internal, size: Small }
+          - { dataset: internal, size: Large }
```
Comment (everpcpc) [inline .github/workflows/reuse.benchmark.yml:104]: no need to test with large

**Comment by sundy-li** [comment]
can we have a separate label `ci-meta-perf` to trigger this benchmark?

**Comment by TCeason** [comment]
> can we have a separate label `ci-meta-perf` to trigger this benchmark?

I think this test belong a internal query test. 

If we need other speed test also can add query in internal/queries.sql. So maybe use same label with tech and hits is ok. They need test together.

File: .github/workflows/reuse.benchmark.yml (hunk 2 above)
Comment (TCeason) [reply .github/workflows/reuse.benchmark.yml:104]: Ok will only contain small size

File: .github/actions/benchmark_cloud/action.yml [hunk 3]
```diff
@@ -55,6 +55,7 @@ runs:
         CLOUD_WAREHOUSE: benchmark-${{ inputs.run_id }}
       shell: bash
       run: |
+        python3 -m pip install --quiet mysql-connector-python==8.0.30
```
Comment (everpcpc) [inline .github/actions/benchmark_cloud/action.yml:58]: redundant line?

**Comment by sundy-li** [comment]
> So maybe use same label with tech and hits is ok. They need test together.

Sometimes we may just test internal tests alone (if you refactor the grants), this could save costs for unnecessary tests.

File: .github/workflows/reuse.benchmark_query_meta.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+name: Benchmark
```
Comment (everpcpc) [inline .github/workflows/reuse.benchmark_query_meta.yml:1]: name should be changed

**Review by everpcpc — APPROVED** [review]

File: .github/workflows/reuse.benchmark_query_meta.yml (hunk 4 above)
Comment (TCeason) [reply .github/workflows/reuse.benchmark_query_meta.yml]: Done

File: .github/workflows/reuse.benchmark_query_meta.yml [hunk 5]
```diff
@@ -0,0 +1,91 @@
+name: Benchmark query meta speed
```
[the remark this answers is not in the document]
Comment (TCeason) [reply .github/workflows/reuse.benchmark_query_meta.yml]: Done

File: .github/workflows/reuse.benchmark_query_meta.yml [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+        timeout-minutes: 30
+        id: benchmark_query_meta
+        with:
+          sha: ${{ inputs.sha }}
+          run_id: ${{ inputs.run_id }}
+          dataset: ${{ matrix.dataset }}
+          source: ${{ inputs.source }}
+          source_id: ${{ inputs.source_id }}
+  benchmark:
+    if: contains(github.event.pull_request.labels.*.name, 'ci-benchmark-suites')
```
Comment (TCeason) [inline .github/workflows/reuse.benchmark_query_meta.yml:93]: cc @BohuTANG @everpcpc  I think we need a new label `ci-benchmark-suites` to trigger this test.

**Comment by sundy-li** [comment]
This pr may be buggy, but we don't know until we merge it, the new benchmark only take effects after merged in main.

**Pull request merged**